Character.h

```diff
@@ -53,22 +53,19 @@ struct Character
         std::cout << getName() << "'s attack damage level has been boosted to " << attackDamage << std::endl;
     }
 
-    void printStats()
-    {
-        std::cout << getName() << "'s stats: " << std::endl;
-        std::cout << getStats(); //make your getStats() use a function from the Utility.h
-        
-        std::cout << std::endl;
-        std::cout << std::endl;
-    }
+    void printStats();
+
+public:
+    int attackDamage;
+
```

Comment:
This was private for a reason
- [ ] revert

Reply:
But then I can't use it.